enhancements/sig-policy/99-policy-placement-strategy/README.md

```diff
@@ -0,0 +1,269 @@
+    EventName     string      `json:"eventName,omitempty"`
+  }
+
+  // PolicyStatus defines the observed state of Policy
+  type PolicyStatus struct {
+    Placement []*Placement                  `json:"placement,omitempty"`
+    Status    []*CompliancePerClusterStatus `json:"status,omitempty"`
+
+    // +kubebuilder:validation:Enum=Compliant;Pending;NonCompliant
+    ComplianceState ComplianceState               `json:"compliant,omitempty"`
+    RolloutStatus   clusterv1alpha1.RolloutStatus `json:"rolloutStatus,omitempty"` // <-- New field
+    Details         []*DetailsPerTemplate         `json:"details,omitempty"`
+  }
+  ```
+
+### Implementation Details/Notes/Constraints
+
+For the `Placement` library, this requires importing at least this package version (the `Placement` library is in the
+`v1alpha1` version):
+
+```
+  open-cluster-management.io/api v0.11.1-0.20230809113658-2b2399b5f6e8
+```
+
+For testing, the `governance-policy-propagator` doesn't currently account for multiple managed clusters. As part of this
+enhancement, the test flows would need to be enhanced (and/or a separate workflow created) that deploys multiple managed
+clusters.
+
+### Risks and Mitigation
+
+- The optional proposal to remove items from the `Policy` CRD here is to prevent bloat and try to slim down the CRD, but
+  updating the CRD could be a breaking change for users so further consideration might need to be made and the updates
+  aren't required.
+- The `Placement` library is relatively new and untested outside of its repo and this implementation leans heavily on
+  its logic. While it works in theory, there could be some tweaks/adjustments as development proceeds, lengthening time
+  for implementation. The phased approach intends to address this to make partial implementation feasible.
+- Having a `LastEvaluated` field in the status is going to increase load on the hub cluster Kubernetes API server that
+  efforts originally sought to reduce by not sending updates when the status hadn't changed. This could be remedied by
+  only sending an updated timestamp if the generation was updated and/or the `LastEvaluated` timestamp is less than the
+  root policy timstamp or (the current behavior) if the status indeed changed.
+
+## Design Details
+
+### Open Questions
+
+1. Should the rollout handler be instantiated with the dynamic watcher instead of the controller client?
+2. Should `RolloutStatus` be reflected per-cluster in the root policy? (Removing `clusterNamespace` could essentially
+   make the CRD the same size as it currently is, remediating concerns around bloat, but the rollout status is also
+   reflected in each replicated policy.)
+3. Is there opposition to removing `Placement.Decisions` from the `status`? (This could alternatively be handled outside
+   of this enhancement.)
+4. Should the `RolloutStrategy` be implemented in the `Policy` CRD instead of `PlacementBinding`?
```

Comment:
I prefer this level.

Reply:
This means it needs to be in PolicySet as well.

Reply:
@mprahl I just realized that "this" was unclear. Are you saying you prefer it being in the `Policy`?